keycloak/007-keycloak-not-running-on-docker-container.md

```diff
@@ -0,0 +1,16 @@
+# Keycloak Troubleshooting
+
+## How to troubleshoot when keycloak is not running on Docker container?
+
+> Keycloak is a third party container, and it is the issue of keycloak container image.
+> Some steps to troubleshoot the issue are :
+
+> 1.In case your local Docker (e.g., memory) may not be enough for the Keycloak image , make sure your docker configuration matches below :
```

Comment:
> 1. In case there is not sufficient resources (e.g. CPU, Memory) in your local Docker for running the Keycloak Docker Image, try increasing the resource limitation based on the required usage of your container image as the following example of the Docker configuration::